### PR TITLE
feat(enabler): change application type for enabler

### PR DESCRIPTION
**Pull request by leungkinghin-ct**
- Change `applicationType` to be `assets` for enabler, which supports CDN to static resources
- Update README.md

### CODE REVIEW

File: connect.yaml [hunk 1]
```diff
@@ -24,9 +24,5 @@ deployAs:
...
   - name: enabler
-    applicationType: service
-    endpoint: /
-    configuration:
-      standardConfiguration:
-        - key: CTP_REGION
-          description: Region of Commercetools project
\ No newline at end of file
+    applicationType: assets
+    endpoint: /
```
Comment (koonweiteo-commercetools) [inline connect.yaml:28]: thanks @leungkinghin , we can remove endpoint too.

Comment (leungkinghin) [reply connect.yaml]: https://github.com/commercetools/connect-payment-integration-template/pull/111/commits/7f0656f959c2711fe25576dbd6e1a0314445b6c8

Comment (abreu-ct) [inline connect.yaml]: For the `assets` app we can also remove the `endpoint` field as it is not required

**Review by abreu-ct — CHANGES_REQUESTED** [review]

File: connect.yaml (hunk 1 above)
Comment (leungkinghin-ct) [reply connect.yaml]: https://github.com/commercetools/connect-payment-integration-template/commit/7f0656f959c2711fe25576dbd6e1a0314445b6c8

Comment (abreu-ct) [reply connect.yaml]: Sorry @leungkinghin-ct, missed that commit

**Review by abreu-ct — APPROVED** [review]

**Review by koonweiteo-commercetools — APPROVED** [review]

File: connect.yaml (hunk 1 above)
Comment (leungkinghin-ct) [reply connect.yaml]: @abreu-ct no worry :)

**Pull request merged**